test/unit/queues.test.ts

```diff
@@ -89,6 +89,9 @@ describe("queues", () => {
                     "Fn::GetAtt": ["EmailsWorkerLambdaFunction", "Arn"],
                 },
                 MaximumBatchingWindowInSeconds: 0,
+                ScalingConfig: {
+                    MaximumConcurrency: 2,
+                },
```

Comment:
This test should not change, because by default things should not change for users.

Instead, the `MaximumConcurrency` setting should be set only if they set it in `serverless.yml`.

I would suggest adding a new test that tests for `MaximumConcurrency`, and the current test here should not be modified. Does that make sense?

Reply:
its the default value for `MaximumConcurrency`, even you do not specify a default, it will use serverless default value thats 2.